### ISSUE TITLE
Can I see the logs of the app that was installed using a dmg on macOS?

### ISSUE THREAD

**Issue by jkulak** [AllToMP3/alltomp3-app#132]
- the operating system you're using (Windows, macOS, Linux); macOS
- the AllToMP3 version you're using; 0.3.11
- the exact URL or searched you entered into AllToMP3: Public Spotify playlist

I installed the app using the ready version from the dmg. I would like to do some debugging without installing node and Angular. Is it possible to see the logs somewhere?

**Comment by tikkerei** [comment]
Yes, I have also tried to find how to get any logs from the app, without any result.
